Translation/Extractor/File/FormExtractor.php

```diff
@@ -138,7 +142,7 @@ public function enterNode(Node $node)
                         $this->parseItem($item, $domain);
                         break;
                     case 'choices':
-                        if ($this->parseChoiceNode($item, $node, $domain)) {
+                        if ($this->parseChoiceNode($item, $node, $choiceDomain)) {
                             continue 2;
```

Comment:
Oh, I missed the -$domain part, I thought it was an addition. You are correct and can ignore this feedback.